massa-bootstrap/Cargo.toml

```diff
@@ -40,6 +36,7 @@ massa_signature = { path = "../massa-signature" }
 massa_pos_exports = { path = "../massa-pos-exports" }
 massa_time = { path = "../massa-time" }
 crossbeam = "0.8.2"
+mio = {version = "0.8.6", features = ["os-poll", "net"]}
```

Comment:
mio && crossbeam should be in dependencies not "custom modules"

Reply:
Thanks for the catch!